package-config.cmake

```diff
@@ -152,6 +206,12 @@ if(_PKG_CONFIG_REQUIRES)
   endforeach()
   list(REMOVE_DUPLICATES _PKG_CONFIG_REQUIRES_LIST)
 endif(_PKG_CONFIG_REQUIRES)
+
+COMPUTE_TARGETS_LIST()
+if(TARGETS_LIST_LENGTH)
+  SET(VALID_TARGETS_EXPORT_NAME "SET(VALID_TARGETS_EXPORT_NAME TRUE)")
```

Comment:
What is the scope of variable `VALID_TARGETS_EXPORT_NAME` in the configured Config.cmake ? Will it interfere with previous included config files ?

Reply:
Good point.